src/ir/block_builder_test.cc

```diff
@@ -20,7 +20,7 @@
 #include "src/common/testing/gtest.h"
 #include "src/ir/attributes/attribute.h"
 #include "src/ir/ir_context.h"
-#include  "third_party/raksha/src/ir/ir_printer.h"
+#include "src/ir/ir_printer.h"
```

Comment:
Does the internal build still work after this change? (e.g. this fixes external, but it still works internally?)

Reply:
I think it will, yes -- internal code is transformed by copybara to add the prefix. The question is how the prefix made it back out, it should have been stripped (again, if needed). I theorize the extra space may have broken the regex. Am talking about it in Raksha chat.